tests/modules/individuals/resources/test_individual_crud.py

```diff
@@ -91,63 +93,133 @@ def test_read_encounter_from_edm(db, flask_app_client):
+    # )
+
+    # owner_1 = utils.generate_user_instance(
+    #     email='owner@localhost',
+    #     is_researcher=True,
+    # )
+
+    data_in = {
+        'startTime': datetime.datetime.now().isoformat() + 'Z',
+        'context': 'test',
+        'locationId': 'test',
+        'encounters': [
+            {},
+            {},
+            {},
+            {'locationId': 'Monster Island'},
+        ],
+    }
+
+    response = sighting_utils.create_sighting(
+        flask_app_client, researcher_1, expected_status_code=200, data_in=data_in
+    )
+
+    # log.warning(
+    #     '********** test_add_remove_encounters TEST RESPONSE ON SIGHTING CREATE: ' + str(response.json)
+    # )
+
+    from app.modules.sightings.models import Sighting
+
+    sighting_id = response.json['result']['id']
+    sighting = Sighting.query.get(sighting_id)
+
+    assert response.json['success']
+    result_data = response.json['result']
+
+    from app.modules.encounters.models import Encounter
+
+    enc_1 = Encounter(
+        guid=result_data['encounters'][0]['id'],
+        owner_guid=researcher_1.guid,
+    )
 
-    mod_enc_1 = utils.generate_encounter_instance(
-        user_email='mod1@user', user_password='mod1user', user_full_name='Test User'
+    enc_2 = Encounter(
+        guid=result_data['encounters'][1]['id'],
+        owner_guid=researcher_1.guid,
     )
-    mod_enc_2 = utils.generate_encounter_instance(
-        user_email='mod2@user', user_password='mod2user', user_full_name='Test User'
+
+    enc_3 = Encounter(
+        guid=result_data['encounters'][2]['id'],
+        owner_guid=researcher_1.guid,
     )
 
-    # You need to own an individual to modify it, and ownership is determined from it's encounters
-    mod_enc_1.owner = researcher_1
-    mod_enc_2.owner = researcher_1
-    empty_individual.add_encounter(mod_enc_1)
+    response = individual_utils.create_individual(
+        flask_app_client, researcher_1, 200, {'encounters': [{'id': str(enc_1.guid)}]}
+    )
+    individual_1 = Individual.query.get(response.json['result']['id'])
 
-    with db.session.begin():
-        db.session.add(empty_individual)
-        db.session.add(mod_enc_1)
-        db.session.add(mod_enc_2)
+    # # let's start with one
+    # individual_1.add_encounter(enc_1)
 
-    assert str(mod_enc_1.guid) in [
-        str(encounter.guid) for encounter in empty_individual.get_encounters()
+    assert str(enc_1.guid) in [
+        str(encounter.guid) for encounter in individual_1.get_encounters()
     ]
 
-    replace_encounters = [
-        utils.patch_replace_op('encounters', [str(mod_enc_2.guid)]),
+    add_encounters = [
+        utils.patch_add_op('encounters', [str(enc_2.guid)]),
     ]
 
+    import json
+
     individual_utils.patch_individual(
         flask_app_client,
-        '%s' % empty_individual.guid,
         researcher_1,
-        replace_encounters,
-        200,
+        '%s' % individual_1.guid,
+        patch_data=json.dumps(add_encounters),
```

Comment:
This is the bug. individual_utils.patch_individual does the json.dumps on the data so this doing it again results in garbage data being sent to houston. 
You have a few oof these patch calls and each one has the same bug that needs to be removed.